bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -304,6 +333,15 @@ data:
       REDIS_MASTER_PORT_NUMBER="${REDIS_EXTERNAL_MASTER_PORT}"
     fi
 
+    cp /opt/bitnami/redis-sentinel/mounted-etc/sentinel.conf /opt/bitnami/redis-sentinel/etc/sentinel.conf
```

Comment:
Both `start-node.sh` (redis container) and `start-sentinel.sh` (sentinel container) are executed in parallel, and `/opt/bitnami/redis-sentinel/etc/sentinel.conf` is mounted in both containers. 

This overwrite could potentially affect `start-node.sh` since we can't ensure lines 112-119 are executed before this, shouldn't we add some way to control that?

Reply:
Added a lock and a loop just in case